### ISSUE TITLE
scripts: GitHub merge script is missing review comments

### ISSUE THREAD

**Issue by fanquake** [bitcoin/bitcoin#15816]
In #15643 the [`github-merge.py`](https://github.com/bitcoin/bitcoin/blob/master/contrib/devtools/github-merge.py) script was updated so that merge commits would include review comments. 

However, the script seems to be missing comments. i.e the recently merged #15557 includes:

```
ACKs for commit 184f87:
      jnewbery:
        utACK 184f8785f710d58d9ef82e611591c9cbff5ab89d
    
    Tree-SHA512: fb6542bdfb2c6010e328ec475cf9dcbff4eb2b1a1b27f78010214534908987a5635797196fa05edddffcbcf2987335872dc644a99261886d5cbb34a8f262ad3e
```

Looking at the PR there should also be `utACKs` from [ryanofsky](https://github.com/bitcoin/bitcoin/pull/15557#pullrequestreview-225616510) and [promag](https://github.com/bitcoin/bitcoin/pull/15557#issuecomment-482985563).

**Comment by ryanofsky** [comment]
Looking at the code around `retrieve_json` in [`github-merge.py`](https://github.com/bitcoin/bitcoin/blob/master/contrib/devtools/github-merge.py), I don't see anything in there that handles pagination.

The code should be checking for `Link: <...> rel="next"` response headers and sending followup requests if it's supposed to download all review comments. See https://developer.github.com/v3/guides/traversing-with-pagination/#basics-of-pagination

**Comment by fanquake** [comment]
Should have been fixed by #15838.
